From 79b4dae670a360aff466d28f2f2142083bdbb558 Mon Sep 17 00:00:00 2001
From: Christian Berendt <berendt@osism.tech>
Date: Wed, 15 May 2024 21:15:54 +0200
Subject: [PATCH] ceilometer: handle pipeline.yaml as template

The ceilometer/pipeline.yaml file should be handled as a template
like the {ceilometer-compute,ceilometer-central,ceilometer-notification}/pipeline.yaml
files.

Change-Id: Ic3245ae2955792521cf1dddcbd4c5c706f7a05a5
---

diff --git a/ansible/roles/ceilometer/tasks/config.yml b/ansible/roles/ceilometer/tasks/config.yml
index 2835ec4..7cd51ab 100644
--- a/ansible/roles/ceilometer/tasks/config.yml
+++ b/ansible/roles/ceilometer/tasks/config.yml
@@ -302,7 +302,7 @@
       - ceilometer-compute
       - ceilometer-central
       - ceilometer-notification
-  copy:
+  template:
     src: "{{ node_custom_config }}/ceilometer/pipeline.yaml"
     dest: "{{ node_config_directory }}/{{ item.key }}/pipeline.yaml"
     force: True
diff --git a/releasenotes/notes/ceilometer-pipeline-yaml-as-template-8319d7fdeacd15f1.yaml b/releasenotes/notes/ceilometer-pipeline-yaml-as-template-8319d7fdeacd15f1.yaml
new file mode 100644
index 0000000..7bf93c0
--- /dev/null
+++ b/releasenotes/notes/ceilometer-pipeline-yaml-as-template-8319d7fdeacd15f1.yaml
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    The ``ceilometer/pipeline.yaml`` file is now handled as a template
+    like the ``{ceilometer-compute,ceilometer-central,ceilometer-notification}/pipeline.yaml``
+    files.
